Jenkinsfile

```diff
@@ -488,6 +499,91 @@ gcloud container clusters create ${clusterName} \
         }
     }
 
+    def eksVersions = ["1.10", "1.11"]
+    for (x in eksVersions) {
+        def kversion = x // local bind required because closures
+        def platform = "eks-${kversion}"
+        platforms[platform] = {
+            stage(platform) {
+                node(label) {
+                    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'aws-kubeprod-jenkins']]) {
```

Comment:
This jenkins secret still needs to be created.

Reply:
Added to Jenkins (and changed the secret ID in Jenkins to a more meaningful name)